data/brands/amenity/fast_food.json

```
@@ -1438,6 +1438,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Goli Vada Pav",
+      "id": "golivadapav-aba431",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "alt_name": "Goli Vadapav No.1",
+        "amenity": "fast_food",
+        "brand": "Goli Vada Pav Pvt. Ltd",
```

Just the simple name is fine here - OSM doesn't really put the legal name here.
```suggestion
        "brand": "Goli Vada Pav",
```